### DISCUSSION TITLE
"Failed" status visable within the Achievements menu

### DISCUSSION THREAD

**Discussion by ghost** [RetroAchievements/RAIntegration#908] [Ideas]

**Comment by Jamiras** [comment]
Most achievements are defined as "In location/level X, value Y changes to Z". For those achievements (which includes most missables), there's nothing to know when the player has passed by where the achievement would be earnable. If they player has reached "level X+1", the logic "level = X" is false, and the achievement won't trigger. It has no concept of "X+1" or "X-1" to know the player's relation to the target requirement.

Challenge achievements are usually defined as "In location/level X, after N happens, value Y changes to Z unless M happens". When N happens in level X, the indicator is shown until Y changes to Z (success) or M happens (failure). The visibility of the challenge indicator should be sufficient to handle this case.

For achievements tied to a challenge indicator, we could do something to flag the achievement as failed after the challenge indicator is hidden if the achievement is not completed, but that's a completely different issue than reaching a point beyond which a missable can no longer be earned.

Please provide an example of an achievement that you think would benefit from this behavior and how you would expect it to behave.
